src/server/enrollment-system.js

```diff
@@ -63,35 +63,97 @@ function veteranToPersonInfo(veteran) {
   };
 }
 
+function zeroPadNumber(number, padding) {
+  return (new Array(padding + 1).join('0') + number).slice(-padding);
+}
+
+function formDateToESDate(dateObject) {
```

Comment:
AFAICT from the xsd, they seem to just be using standard XML `xs:date`...which doesn't quite match what you have here so I'm wondering where it's from.